client/lib/jetpack/wpcom-upsell-controller.tsx

```diff
@@ -37,12 +34,10 @@ export default function upsellSwitch(
 		}
 
 		const isAtomic = isSiteAutomatedTransfer( getState(), siteId );
-		const isEligibleForAT = isEligibleForAutomatedTransfer( getState(), siteId );
 		// Show the Automated Transfer for sites with a Business plan that are not
-		// Atomic sites. These sites must be eligible as well.
-		if ( isBusiness && ! isAtomic && isEligibleForAT ) {
-			const primaryProduct = context.path.includes( '/backup/' ) ? 'backup' : 'scan';
-			context.primary = <BusinessATComponent product={ primaryProduct } />;
+		// Atomic sites.
+		if ( isBusiness && ! isAtomic ) {
```

Comment:
`isAtomic` isn't needed here, and `isBusiness` does not handle e-comm plans. Check out my PR: #43412